ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/highLevelHumanoidControl/highLevelStates/walkingController/states/SingleSupportState.java

```diff
@@ -39,7 +39,7 @@ public SingleSupportState(WalkingStateEnum singleSupportStateEnum,
       this.supportSide = singleSupportStateEnum.getSupportSide();
       swingSide = supportSide.getOppositeSide();
 
-      minimumSwingFraction.set(0.5);
+      minimumSwingFraction.set(0.6);
```

Comment:
prevents too early touchdown, which happens in the joint based contact switch, because the torques can be high and the velocity is low at the apex of the swing.